### DISCUSSION TITLE
Sync a fork branch with the upstream repository API gives 401 Bad credentials error

### DISCUSSION THREAD

**Discussion by schandra13** [community/community#40643] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I'm trying to incorporate this [API ](https://docs.github.com/en/rest/branches/branches?apiVersion=2022-11-28#sync-a-fork-branch-with-the-upstream-repository)within Azure devops pipeline. The curl command to REST API requires a token to be supplied and I have tried supplying both PAT token from GitHub and System.AccessToken. PAT works for my local and returns a 200 response but doesn't work in the pipeline. What am I doing wrong here? Any suggestions would help.

**Comment by ossanna16** [comment]
Hi there @schandra13 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
